.github/workflows/deploy-ecs.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy to Dev
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        required: true
+        default: dev
+  push:
+    branches: ["main"]
+
+jobs:
+  call-workflow:
+    uses: mbta/workflows/.github/workflows/deploy-ecs.yml@v2
```

Comment:
question(non-blocking): what benefit do we get from pinning this dependency to `@v2` if this is an internal resource?

Reply:
@firestack While we don't really own workflows, Infra is in favor of pinning versions. That way, it's easier for us to roll out changes in a measured and structured fashion, like we did when moving to assumed roles and away from long-lived access keys for workflows.